docs/digger/installation/macos-setup.asciidoc

```diff
@@ -1,59 +0,0 @@
-=== Setup macOS Servers
```

Comment:
Why are we removing this file?

Reply:
this should not have been removed... great catch.

Reply:
i think this has to do something with osx-provision being called off in the past... not sure tho

Reply:
resolved